### ISSUE TITLE
Training Loss: nan

### ISSUE THREAD

**Issue by uridah** [jazzsaxmafia/Weakly_detector#12]
I tried training a model on this but I keep getting 'Training Loss: nan'
I trained for 17 epochs before I stopped because of this.
Is it that the model is not learning or could it be a different issue?
